lib/linguist/heuristics.yml

```
@@ -98,6 +98,12 @@ disambiguations:
   rules:
   - language: FreeBasic
     pattern: '^[ \t]*#(?:define|endif|endmacro|ifn?def|if|include|lang|macro)\s'
+- extensions: ['.bs']
+  rules:
+  - language: Bikeshed
+    pattern: '^(?i:<\/*pre>)|(?i:<pre class=.*>)'
```

According to [Bikeshed's spec](https://tabatkins.github.io/bikeshed/#metadata), several metadata fields are required information, stipulating the presence of a `<pre class="metadata">…</pre>` block. We can therefore afford to make this pattern more specific (and ergo, less prone to false matches).

```suggestion
    pattern: '^(?i:<pre\s+class)\s*=\s*('|"|\b)?metadata\1[^>\r\n]*>'
```